common/models/user.js

```diff
@@ -752,10 +762,10 @@ module.exports = function(User) {
       {
         description: 'Logout a user with access token.',
         accepts: [
-          {arg: 'access_token', type: 'string', required: true, http: function(ctx) {
```

Comment:
With `required:true`, logout without any access tokens returns 400 because strong-remoting rejects the request as missing required arguments.